packages/Overlay/src/Overlay.js

```diff
@@ -5,8 +5,7 @@ import FocusLock from "react-focus-lock";
 import LockBodyScroll from "@paprika/helpers/lib/components/LockBodyScroll";
 import Portal from "@paprika/helpers/lib/components/Portal";
 import tokens from "@paprika/tokens";
-import Wrapper from "./Wrapper";
-import Backdrop from "./Backdrop";
+import * as sc from "./Overlay.styles";
```

Comment:
Aligning with our more current practice of exporting styled components from a `.styles.js` file and importing all to a `sc` object.